sdk/typescript/src/cryptography/secp256k1-keypair.ts

```diff
@@ -0,0 +1,123 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import * as secp from "@noble/secp256k1";
```

Comment:
https://www.npmjs.com/package/@noble/secp256k1 
- this is a top 10 library for secp256k1
- the benchmark marks itself to be the fastest, esp signing
- I liked this one the most bc 0 dep
- the latest version is less than 6 months old

here are some other ones, https://www.npmjs.com/search?ranking=optimal&q=secp256k1

another option is https://www.npmjs.com/package/secp256k1 a native C binding to the underlying library (this may be nicer since our rust library also wraps to the same C library)

Reply:
+ audited